components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -142,8 +161,12 @@ export default function () {
 
             {showPersistentVolumeClaimUI && (
                 <>
-                    <br></br>
-                    <h3 className="mt-12">Workspaces</h3>
+                    <SelectWorkspaceClass
```

Comment:
**thought**: Feels good to see we're working towards making Teams & Projects more useful.